doc/client_lua_api.md

```diff
@@ -648,10 +648,10 @@ Call these functions only at load time!
       semi-frequent intervals as well as on server shutdown.
 * `minetest.register_on_connect(func())`
     * Called at the end of client connection (when player is loaded onto map)
-* `minetest.register_on_receiving_chat_message(func(name, message))`
+* `minetest.register_on_receiving_chat_messages(func(message))`
```

Comment:
This function is called for each new message. It would be better to correct the spelling in `s_client.cpp`, as only one message will be sent in argument 1. An additional line in the builtin code will keep the backwards compat: `core.register_on_receiving_chat_messages = core.register_on_receiving_chat_message`